test/integration/test_spark_lightning.py

```diff
@@ -119,6 +130,10 @@ def __init__(self, *args, **kwargs):
         warnings.simplefilter('module')
 
     def test_fit_model(self):
+        if skip_lightning_tests:
+            self.skipTest('Spark PyTorch Lightning tests conflict with Tensorflow 2.5.x: '
+                          'https://github.com/horovod/horovod/pull/3263')
+
```

Comment:
It looks as if such a block might be missing for `test_terminate_on_nan_flag()` right now. Could the skip condition be integrated into the 
```
@pytest.mark.skipif(LooseVersion(pl.__version__) < LooseVersion(MIN_PL_VERSION), reason='Pytorch lightning version is not supported.')
class SparkLightningTests(unittest.TestCase):
```
decorator otherwise?

Reply:
> It looks as if such a block might be missing for `test_terminate_on_nan_flag()` right now.

`test_terminate_on_nan_flag()` does not fail

> Could the skip condition be integrated into the ...

Sure, but then all tests would be skipped, half of the tests do not fail.

Reply:
I see, thanks!